### PR TITLE
323 canada fed rename

### LINKED ISSUE #323: Using district names in Canadian Federal Electoral District OCD-IDs
A new set of OCD-IDs are created every time there is redistricting creating a problem for how we represent districts contiguously across time. Rather than use the numeric federal district ids, which only represent a district’s identity for 10 years, we should use district names as they are more closely represent a district’s identity. Some concerns regarding the switch to using names as identifiers are that they can change between redistricting cycles (see https://www12.statcan.gc.ca/census-recensement/2011/ref/FED_name_changes-changements_noms_CEF-eng.cfm and https://openparliament.ca/bills/42-1/C-402/). When names are changed, new ids and aliases can be created so that identity is maintained through the canonical OCD-ID. When redistricting occurs, districts created after redistricting that don’t map to an old one (not the same name or renaming of a district) will be considered new districts and ids will be created for them. Old districts which no longer exist will have a validThrough date to indicate it is no longer active.

### PR DESCRIPTION
**Pull request by evannjw**
Cleaned up conflicting aliases and completed TODOs on draft pr.
For missing sameAs, these were moved to abolished districts csv as there is no clear source to use to address these.

closes #323 (which followed from #320)
replaces #324

### CODE REVIEW

**Comment by evannjw** [comment]
@jpmckinney Can you take a look?

File: identifiers/country-ca/README.md
Comment (jloutsenhizer) [inline identifiers/country-ca/README.md:1]: It might be nice to add to this a list of files which are generated by scripts vs manually maintained

**Comment by evannjw** [comment]
@jpmckinney Hi sorry to ping again, would you be able to review in the next week or two?

**Review by jloutsenhizer — APPROVED** [review]

**Comment by jpmckinney** [comment]
I'll try this weekend!

**Comment by jpmckinney** [comment]
I haven't had time to review, but if this is time-sensitive, happy for another reviewer to merge.

**Comment by evannjw** [comment]
@DrMeers Would you be able to review and merge?

**Comment by DrMeers** [comment]
> @DrMeers Would you be able to review and merge?

Unfortunately I lack both the knowledge and the permissions to do so

**Comment by jloutsenhizer** [comment]
@opencivicdata/division-id-curators could someone provide a second review for this PR? Thanks

**Comment by evannjw** [comment]
This has been sitting around for while now, any chance you could review this sometime this/next week @jpmckinney?

**Comment by jpmckinney** [comment]
It doesn't look like I'll have time to review properly, so feel free to merge as-is. I expect that at most there might be a few individual corrections. My projects will continue to use aliases for now, as I don't have time to transition to new IDs.

**Comment by NikolasYo** [comment]
Thanks for the ping. As we reviewed with @zikowang it seems okay.

**Comment by zikowang** [comment]
@evannjw @jpmckinney this branch was merged into https://github.com/opencivicdata/ocd-division-ids/tree/323-canada-fed-rename, is it correct? Shouldn't it be merged into `master`?

Ich would continue woking on #364 after the merge.

**Comment by jpmckinney** [comment]
Ah, yes, I hadn't noticed that.

**Comment by jpmckinney** [comment]
I opened a new PR. Can you resolve the conflicts in #365?

**Pull request merged**
